.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Thank you for contributing to The Foreman, please read the following guide
+available at https://www.theforeman.org/contribute.html - in short:
+
+* Create RedMine issue at http://projects.theforeman.org/projects/foreman
+* Reference the issue via "Fixes #1234" or "Refs" in the commit message
+* Run Rubocop prior sending your PR, check Coding Standards in the guide
+* Extract all strings for l18n, see Translation section in the guide
+* Prepend `[WIP]` or `WIP` to prevent lengthy tests for work in progress
+* Be patient, we will do our best to take a look as soon as we can.
+* Explain what you are traing to do. Be nice and respectful.
```

Comment:
s/traing/trying/

Reply:
Reworded to "Explain purpose of the PR" which seems to be cleaner.